keyboards/pica40/keymaps/default/keymap.c

```diff
@@ -0,0 +1,212 @@
+        KC_MINS, KC_Z,    KC_X,    KC_C,    KC_V,    KC_B,                KC_N,    KC_M,    KC_COMM, KC_DOT,  KC_SLSH,   KC_GRV,
+                                      FUNC_ESC, NUM_TAB, SYM_SPC,     FUNC_ENT, NAV_BSPC, RALT_DEL
+    ),
+
+    [_GAME] = LAYOUT(
+                   KC_Q,   KC_W,    KC_E,    KC_R,    KC_T,               KC_Y,    KC_U,    KC_I,    KC_O,    KC_P,
+        LALT_EQL,  KC_A,   KC_S,    KC_D,    KC_F,    KC_G,               KC_H,    KC_J,    KC_K,    KC_L,    KC_SCLN, LGUI_QUOT,
+        LSFT_MINS, KC_Z,   KC_X,    KC_C,    KC_V,    KC_B,               KC_N,    KC_M,    KC_COMM, KC_DOT,  KC_SLSH, TG(_GAME),
+                                       LCTL_ESC, NUM_TAB, SYM_SPC,    FUNC_ENT, NAV_BSPC, RALT_DEL
+    ),
+
+    [_NAV] = LAYOUT(
+                 XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX,             XXXXXXX,       XXXXXXX, KC_PGDN, KC_PGUP, XXXXXXX,
+        XXXXXXX, KC_LGUI, KC_LALT, KC_LCTL, KC_LSFT, XXXXXXX,             LALT(KC_UP),   KC_LEFT, KC_DOWN, KC_UP,   KC_RGHT, KC_INS,
+        XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX,             LALT(KC_DOWN), KC_HOME, KC_END,  KC_APP,  XXXXXXX, XXXXXXX,
+                                       XXXXXXX, XXXXXXX, XXXXXXX,     XXXXXXX, KC_TRNS, XXXXXXX
+    ),
+
+    [_NUMBER] = LAYOUT(
+                 KC_BSLS, KC_7,    KC_8,    KC_9,    KC_0,                XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX,
+        KC_LCTL, KC_COMM, KC_4,    KC_5,    KC_6,    KC_LBRC,             XXXXXXX, KC_RSFT, KC_RCTL, KC_LALT, KC_RGUI, XXXXXXX,
+        KC_DEL,  KC_DOT,  KC_1,    KC_2,    KC_3,    KC_RBRC,             XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX,
+                                      KC_SPC, KC_TRNS, XXXXXXX,       XXXXXXX, XXXXXXX, XXXXXXX
+    ),
+
+    [_SYMBOL] = LAYOUT(
+                 LSFT(KC_BSLS), LSFT(KC_7), LSFT(KC_8), LSFT(KC_9), LSFT(KC_0),               XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX,
+        KC_LCTL, LSFT(KC_COMM), LSFT(KC_4), LSFT(KC_5), LSFT(KC_6), LSFT(KC_LBRC),            XXXXXXX, KC_RSFT, KC_RCTL, KC_LALT, KC_RGUI, XXXXXXX,
+        KC_BSPC, LSFT(KC_DOT),  LSFT(KC_1), LSFT(KC_2), LSFT(KC_3), LSFT(KC_RBRC),            XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX,
+                                                    LSFT(KC_GRV), TG(_GAME), KC_TRNS,     XXXXXXX, XXXXXXX, XXXXXXX
+    ),
+
+    [_FUNC] = LAYOUT(
+                 KC_F12, KC_F7,   KC_F8,   KC_F9,   KC_PSCR,            XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX,
+        KC_LCTL, KC_F11, KC_F4,   KC_F5,   KC_F6,   KC_PAUS,            XXXXXXX, KC_RSFT, KC_RCTL, KC_LALT, KC_RGUI, XXXXXXX,
+        KC_ENT,  KC_F10, KC_F1,   KC_F2,   KC_F3,   KC_CAPS,            XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX,
+                                     KC_TRNS, KC_MNXT, KC_MPLY,     KC_TRNS, XXXXXXX, XXXXXXX
+    )
+};
+
+bool get_tapping_force_hold(uint16_t keycode, keyrecord_t *record) {
```

Comment:
The `default` keymap generally shouldn't contain any custom code like what's below. This should be removed or moved to another keymap.

Reply:
Moved that keymap into a new non-default one, added new `default` without any code.